docs/source/analyze/velero.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Velero
+description: Check backup, restore settings and status.
+---
+
+The `Velero` analyzer is available to check statuses of Custom Resources installed by velero, such as: backup storage locations, backup repositories, backups and restores.
+
+## Parameters
+
+**namespace:** (Optional) Namespace where velero is installed. Will default to `velero` if not set.
```

Comment:
If we chose to remove this in https://github.com/replicatedhq/troubleshoot/pull/1366#pullrequestreview-1710845118, reflect the change here